package.json

```diff
@@ -1,26 +1,321 @@
+    "normalize-package-data": "^3.0.3",
+    "once": "^1.4.0",
+    "path-exists": "^4.0.0",
+    "parent-module": "^1.0.1",
+    "path-key": "^3.1.1",
+    "p-locate": "^4.1.0",
+    "parse-json": "^5.2.0",
+    "p-try": "^2.2.0",
+    "pkg-dir": "^4.2.0",
+    "path-type": "^4.0.0",
+    "path-is-absolute": "^1.0.1",
+    "pirates": "^4.0.5",
+    "path-parse": "^1.0.7",
+    "picocolors": "^1.0.0",
+    "prompts": "^2.4.2",
+    "q": "^1.5.1",
+    "pretty-format": "^29.1.2",
+    "picomatch": "^2.3.1",
+    "readable-stream": "^3.6.0",
+    "react-is": "^18.2.0",
+    "punycode": "^2.1.1",
+    "read-pkg-up": "^7.0.1",
+    "quick-lru": "^4.0.1",
+    "read-pkg": "^5.2.0",
+    "require-directory": "^2.1.1",
+    "redent": "^3.0.0",
+    "resolve-from": "^5.0.0",
+    "require-from-string": "^2.0.2",
+    "resolve-global": "^1.0.0",
+    "regex": "^0.1.1",
+    "shebang-command": "^2.0.0",
+    "signal-exit": "^3.0.7",
+    "semver": "^6.3.0",
+    "shebang-regex": "^3.0.0",
+    "resolve": "^1.22.1",
+    "resolve.exports": "^1.1.0",
+    "safe-buffer": "^5.2.1",
+    "slash": "^3.0.0",
+    "sorted-array": "^1.1.0",
+    "sisteransi": "^1.0.5",
+    "resolve-cwd": "^3.0.0",
+    "source-map-support": "^0.5.13",
+    "source-map": "^0.6.1",
+    "split2": "^3.2.2",
+    "spdx-exceptions": "^2.3.0",
+    "spdx-correct": "^3.1.1",
+    "spdx-expression-parse": "^3.0.1",
+    "spdx-license-ids": "^3.0.12",
+    "stack-utils": "^2.0.5",
+    "string-width": "^4.2.3",
+    "strip-ansi": "^6.0.1",
+    "string-length": "^4.0.2",
+    "strip-final-newline": "^2.0.0",
+    "sprintf-js": "^1.0.3",
+    "supports-color": "^7.2.0",
+    "statemachines": "^0.1.0",
+    "strip-indent": "^3.0.0",
+    "strip-bom": "^4.0.0",
+    "string_decoder": "^1.3.0",
+    "text-extensions": "^1.9.0",
+    "supports-hyperlinks": "^2.3.0",
+    "through2": "^4.0.2",
+    "test-exclude": "^6.0.0",
+    "strip-json-comments": "^3.1.1",
+    "terminal-link": "^2.1.1",
+    "through": "^2.3.8",
+    "supports-preserve-symlinks-flag": "^1.0.0",
+    "to-fast-properties": "^2.0.0",
+    "ts-node": "^10.9.1",
+    "tmpl": "^1.0.5",
+    "trim-newlines": "^3.0.1",
+    "type-detect": "^4.0.8",
+    "to-regex-range": "^5.0.1",
+    "universalify": "^2.0.0",
+    "typescript-dotnet-commonjs": "^4.11.2",
+    "typescript": "^4.8.4",
+    "type-fest": "^0.21.3",
+    "uri-js": "^4.4.1",
+    "ts-jest": "^29.0.3",
+    "util-deprecate": "^1.0.2",
+    "underscore": "^1.1.7",
+    "which": "^2.0.2",
+    "update-browserslist-db": "^1.0.10",
+    "wrap-ansi": "^7.0.0",
+    "v8-to-istanbul": "^9.0.1",
+    "y18n": "^5.0.8",
+    "yn": "^3.1.1",
+    "yallist": "^4.0.0",
+    "yargs": "^17.6.0",
+    "v8-compile-cache-lib": "^3.0.1",
+    "validate-npm-package-license": "^3.0.4",
+    "yargs-parser": "^21.1.1",
+    "write-file-atomic": "^4.0.2",
+    "yocto-queue": "^0.1.0",
+    "walker": "^1.0.8",
+    "wrappy": "^1.0.2",
+    "@commitlint/config-conventional": "^17.1.0"
```

Comment:
@realmarv this doesn't look correct, the title of this PR states version 17.4.0, not 17.1.0

Reply:
This is one of the commitlint dependencies.  Commitlint's version is 17.4.0. But I can upgrade this one as well. I hope it doesn't break the CI

Reply:
>  I hope it doesn't break the CI

Try it in different branch first?

Reply:
this is still not addressed ^ pls do next day

Reply:
Tried here: https://github.com/realmarv/conventions/commits/config-conventional-17.4.0
It seems ok

Reply:
Do I apply the changes here?